src/lib/util/Colors.js

```diff
@@ -90,7 +90,7 @@ class Colors {
 	/**
 	 * Apply the text format
 	 * @since 0.5.0
-	 * @param {ColorsFormatType} text The text format to apply
+	 * @param {?string} text The text format to apply
```

Comment:
JSDocs doesn't use `?` to denote optional
```suggestion
	 * @param {string} [text] The text format to apply
```